### PR TITLE
updated Romanian translation

### PR DESCRIPTION
**Pull request by pmagixc**
title speaks for itself

### CODE REVIEW

File: Profile Folder/chrome/locale/ro/dtd/gsettings.dtd [hunk 1]
```diff
@@ -90,10 +90,10 @@
 <!ENTITY gsettings.enableWebsitePinningInEveryEra				"Activați fixarea site-ului în fiecare temă de Chrome in Geckium">
 <!ENTITY gsettings.enableWebsitePinningInEveryEraDescription	"Afișează butonul pentru fixare în fiecare design">
 
-<!ENTITY gsettings.about        "About">
-<!ENTITY gsettings.chooseAbout  "Choose About">
-<!ENTITY gsettings.browser      "Browser">
-<!ENTITY gsettings.ask          "Ask">
+<!ENTITY gsettings.about        "Despre">
+<!ENTITY gsettings.chooseAbout  "Selectează afișarea secțiunei ”despre„">
```
Comment (angelbruni) [inline Profile Folder/chrome/locale/ro/dtd/gsettings.dtd:94]: Are you having trouble using quotes or is that intentional?

**Review by angelbruni — COMMENTED** [review]
Sorry for taking so long, I've been sick for these past few days.

File: Profile Folder/chrome/locale/ro/dtd/gsettings.dtd (hunk 1 above)
Comment (pmagixc) [reply Profile Folder/chrome/locale/ro/dtd/gsettings.dtd:94]: those are the quotes used in Romania

Comment (pmagixc) [reply Profile Folder/chrome/locale/ro/dtd/gsettings.dtd:94]: tho i think ive made a mistake, thank you for pointing it out now lol

**Comment by angelbruni** [comment]
Thanks

**Pull request merged**